modules/solid_properties/test/tests/materials/tungsten/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
```

Comment:
I think you should add a test for when the class is used outside the range of definition of the properties.